addons/mail/static/src/messaging/entity/messaging_initializer/messaging_initializer.js

```diff
@@ -178,15 +170,19 @@ function MessagingInitializerFactory({ Entity }) {
             },
             moderation_channel_ids = [],
         }) {
+            // AKU FIXME: before any creation of channel/thread because this is
+            // used for computed field `isModeratedByUser`.
+            this.env.entities.Thread.moderatedChannelIds = moderation_channel_ids;
             for (const data of channel_channel.concat(channel_direct_message, channel_livechat, channel_private_group)) {
-                let channel = this.env.entities.Thread.channelFromId(data.id);
-                if (!channel) {
-                    channel = this.env.entities.Thread.create(
-                        Object.assign({}, data, { isPinned: true })
-                    );
-                }
+                this.env.entities.Thread.insert(Object.assign(
+                    {},
+                    this.env.entities.Thread.convertData(data),
+                    {
+                        isPinned: true,
+                        model: 'mail.channel',
+                    }
+                ));
```

Comment:
strange indent, missing, in last object

Reply:
Yep, I got tricked on the `,` because of the indent... I also said it on comments after this one but I removed them when I noticed.